frontend/app/.eslintrc-auto-import.json

```diff
@@ -240,6 +240,7 @@
     "toSortedAssetBalanceArray": true,
     "toSortedAssetBalanceWithPrice": true,
     "toUnit": true,
+    "toValue": true,
```

Comment:
What is this? Why did you make this change? Why did you make these changes to the frontend files?!?

Reply:
This is weird, removed it

Reply:
This happened because @nebolax didn't do `pnpm install` and his node_modules has an older version of `@vueuse`